ConwaysGameOfLife/Universe.cs

```diff
@@ -48,15 +48,21 @@ public bool AreAllCellsDead()
 
         public List<Location> GetCellNeighbourLocations(Cell cell)
         {
+            var xMiddle = cell.Location.X;

```

Comment:
suggestion: maybe call this plain x and y

Reply:
we paired on this, I mentioned `xMiddle` could be more explicit than `x` if you want to avoid potential confusion. I have no preference personally